op-supervisor/supervisor/backend/backend.go

```diff
@@ -493,7 +497,8 @@ func (su *SupervisorBackend) UpdateLocalSafe(ctx context.Context, chainID types.
 }
 
 func (su *SupervisorBackend) UpdateFinalizedL1(ctx context.Context, chainID types.ChainID, finalized eth.BlockRef) error {
-	return su.chainDBs.UpdateFinalizedL1(finalized)
+	//return su.chainDBs.UpdateFinalizedL1(finalized)
+	return nil
 }
```

Comment:
PR looks good, but this makes me hesitant to merge it. We need better testing in place, and wrap up refactoring with the data-flow changes.

Reply:
I agree and can work on that today. There's a couple things we could do:

1) delete this function (or stop exporting it) so that the op-node can't use it, and chase down the changes in the node. If your concern is that an OP Node might get the `return nil` and misbehave due to misunderstanding of the signal, this would be the fix.

2) The E2E tests use an L1 source, so they have a running L1 Processor, and therefore should be admissible testing for Finality. I don't think we have a finality test in the interop tests yet, I'll check and expand on what we have.

I'll start with E2E tests since that should help us observe correct behavior, and we can go all the way with disconnecting `UpdateFinalizedL1` after that.

Reply:
@protolambda I've now also stacked a second PR which removes the `UpdateFinalizedL1` API from the node and Supervisor altogether. It was a very clean extraction, though the node still manages its own finality (we won't be able to change that until the Supervisor sends heads down to the sync nodes).

This removal is also clean for external nodes who don't direclty rely on a supervisor anymore, as they *also* won't call any update functions anymore.

If this makes merging difficult because you are also working in this area, LMK and I can pull this commit off and put it somewhere else.